Tasks/AndroidSigningV3/task.json

```diff
@@ -0,0 +1,165 @@
+        },
+        {
+            "name": "apksign",
+            "type": "boolean",
+            "label": "Sign the APK",
+            "defaultValue": "true",
+            "required": false,
+            "groupName": "apksignerOptions",
+            "helpMarkDown": "Select this option to sign the APK with a provided keystore file. Unsigned APKs can only run in an emulator. APKs must be signed to run on a device."
+        },
+        {
+            "name": "keystoreFile",
+            "aliases": ["apksignerKeystoreFile"],
+            "type": "secureFile",
+            "label": "Keystore file",
+            "defaultValue": "",
+            "required": true,
+            "visibleRule": "apksign = true",
+            "groupName": "apksignerOptions",
+            "helpMarkDown": "Select the keystore file that was uploaded to `Secure Files` to be used to sign the APK."
+        },
+        {
+            "name": "keystorePass",
+            "aliases": ["apksignerKeystorePassword"],
+            "type": "string",
+            "label": "Keystore password",
+            "defaultValue": "",
+            "required": false,
+            "visibleRule": "apksign = true",
+            "groupName": "apksignerOptions",
+            "helpMarkDown": "Enter the password for the provided keystore file. Use a new variable with its lock enabled on the Variables tab to encrypt this value."
+        },
+        {
+            "name": "keystoreAlias",
+            "aliases": ["apksignerKeystoreAlias"],
+            "type": "string",
+            "label": "Alias",
+            "defaultValue": "",
+            "required": false,
+            "visibleRule": "apksign = true",
+            "groupName": "apksignerOptions",
+            "helpMarkDown": "Enter the alias that identifies the public/private key pair to be used in the keystore file."
+        },
+        {
+            "name": "keyPass",
+            "aliases": ["apksignerKeyPassword"],
+            "type": "string",
+            "label": "Key password",
+            "defaultValue": "",
+            "required": false,
+            "visibleRule": "apksign = true",
+            "groupName": "apksignerOptions",
+            "helpMarkDown": "Enter the key password for the alias and keystore file. Use a new variable with its lock enabled on the Variables tab to encrypt this value."
+        },
+        {
+            "name": "apksignerArguments",
+            "type": "string",
+            "label": "apksigner arguments",
+            "defaultValue": "--verbose",
+            "required": false,
+            "visibleRule": "apksign = true",
+            "groupName": "apksignerOptions",
+            "helpMarkDown": "Provide any options to pass to the apksigner command line. Default is: -verbose -sigalg MD5withRSA -digestalg SHA1"
+        },
+        {
+            "name": "apksignerLocation",
+            "aliases": ["apksignerFile"],
+            "type": "string",
+            "label": "Apksigner location",
```

Comment:
Should the label say "APK signer location?"

Reply:
The tool is called 'apksigner', maybe I shouldn't capitalize the first letter?

Reply:
I think so, follow same pattern as in the android docs.